homeassistant/components/zwave_js/__init__.py

```diff
@@ -358,6 +358,10 @@ def async_on_node_removed(event: dict) -> None:
                 hass,
                 f"{DOMAIN}_{client.driver.controller.home_id}.{node.node_id}.node_status_remove_entity",
             )
+            async_dispatcher_send(
+                hass,
+                f"{DOMAIN}_{client.driver.controller.home_id}.{node.node_id}.ping_remove_entity",
```

Comment:
Maybe we should change the signal and only include the node id and a suffix that is unique for the type of callback, ie remove. Ie make it so that we only need one signal for this operation.